styles.css

```diff
@@ -0,0 +1,265 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,700&display=swap');
```

Comment:
This and normalize would be better off in a styles directory to keep the root of the project clean

Reply:
right, changed